docs/src/dev_guide/inputs.md

```diff
@@ -0,0 +1,337 @@
+# Inputs
+
+Inputs used to run the holistic electricity model: [indices](#indices) and [parameters](#parameters). [Indices](#indices) refer to the dimensions using which the rest of the data is organized, such as the number of generation and storage resources, independent power producers, the balancing areas condsidered, the representative days considered etc.
+
+## Indices
+
+### Common Indices
+
+ - **index_y.csv**: Representative simulation years. It consisits of a single row of year values. The file is used to generate two types of indices, `index_y` which is the years for a simulation window and `index_y_fix` which represents the full simulation horizon (does not change).
+
+ - **index_s.csv**: New resource depreciation years represented as a row vector.
```

Comment:
```suggestion
 - **index_s.csv**: Years from initial build over which new resources depreciate. For example, this could be a list of integers 1, 2, ..., 15.
```